apps/cms/contacts/models.py

```diff
@@ -26,14 +27,15 @@ class FormField(AbstractFormField):
 class WagtailCaptchaFormBuilder(FormBuilder):
     @property
     def formfields(self):
-        # Add captcha to formfields property
         fields = super().formfields
-        fields["captcha"] = CaptcheckCaptchaField(
-            label=_("I am not a robot"),
-            help_text=_(
-                "If you are having difficulty please contact" "us, details adjacent"
-            ),
-        )
+        # Add captcha to formfields property if the URL exists in settings
+        if settings.CAPTCHA_URL:
```

Comment:
I think it is because here the field is added, and in the other form the field is deleted if setting not there. Does that make sense?

Reply:
When I started working on this, there was no CAPTCHA_URL as default with an empty string, thus I did the check for the attribute. Katharina added that url as empty string in the base settings afterwards and while I was away. So technically we don't need the hasattr check any longer. But it can also be there as an extra safety. Let me know what you prefer and I will update either check to be the same.

Reply:
@mara ah, makes sense. I'm fine with both, so pick what you prefer

Reply:
@goapunk @Rineee ok now they are uniformed, and I kept the extra hasattr as extra check in case a project would like to remove the setting. I rebased and added a changelog.